utils/daos_pylint.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+"""Wrapper script for calling pylint"""
+
+from pylint.lint import Run
+from pylint.reporters.collecting_reporter import CollectingReporter
+from pylint.lint import pylinter
+from collections import Counter
+import subprocess  # nosec
+import sys
+import sl.check_script
+
+
+def parse_file(target_file):
+    """Main program"""
+
+    rep = CollectingReporter()
+
+    wrapper = None
+
+    if isinstance(target_file, list):
+        target = list(target_file)
+        target.extend(['--jobs', '100'])
+    elif target_file.endswith('SConstruct') or target_file.endswith('SConscript'):
+        wrapper = sl.check_script.WrapScript(target_file)
+        target = [wrapper.wrap_file]
+    else:
+        target = [target_file]
+
+    target.extend(['--persistent', 'n'])
+
+    results = Run(target, reporter=rep, do_exit=False)
+
+    types = Counter()
+    symbols = Counter()
+
+    for msg in results.linter.reporter.messages:
+        vals = {}
+
+        # Spelling mistake, do not complain about message tags.
+        if msg.msg_id in ('C0401', 'C0402'):
+            if ":avocado:" in msg.msg:
+                continue
+
+        if wrapper:
+            vals['path'] = target_file
+            vals['line'] = wrapper.convert_line(msg.line)
+        else:
+            vals['path'] = msg.path
+            vals['line'] = msg.line
+        vals['column'] = msg.column
+        vals['message-id'] = msg.msg_id
+        vals['message'] = msg.msg
+        vals['symbol'] = msg.symbol
+
+        print('{path}:{line}:{column}: {message-id}: {message} ({symbol})'.format(**vals))
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string